src/Orchard.Web/Modules/Orchard.Projections/Services/IProjectionManager.cs

```diff
@@ -17,9 +18,8 @@ public interface IProjectionManager : IDependency {
         SortCriterionDescriptor GetSortCriterion(string category, string type);
         LayoutDescriptor GetLayout(string category, string type);
         PropertyDescriptor GetProperty(string category, string type);
-
-        IEnumerable<ContentItem> GetContentItems(int queryId, int skip = 0, int count = 0);
-        int GetCount(int queryId);
+        IEnumerable<ContentItem> GetContentItems(int queryId, int skip = 0, int count = 0, Dictionary<string, object> tokens = null);
+        int GetCount(int queryId, Dictionary<string, object> tokens = null);

```

Comment:
Please, no `GetCount` with tokens. Assume it's always `null` if an inner call needs it.